crates/primitives/src/l1.rs

```diff
@@ -237,7 +237,7 @@ pub struct L1Status {
 /// some useful traits on it such as [`serde::Deserialize`], [`borsh::BorshSerialize`] and
 /// [`borsh::BorshDeserialize`].
 // TODO: implement [`arbitrary::Arbitrary`]?
-#[derive(Debug, Clone, Serialize, PartialEq, Eq)]
+#[derive(Debug, Clone, Serialize, Eq)]
```

Comment:
Why you implemented `PartialEq` manually and still left `Eq` as derived?
Is it the case that the derived `Eq` will use the `PartialEq` manual impl?

Reply:
Yes, Eq is there to imply equivalence relation and implies PartialEq.